python/feathub/processors/flink/table_builder/sliding_window_utils.py

```diff
@@ -199,6 +212,14 @@ def evaluate_sliding_window_transform(
             ],
         )
 
+    if config.get(ENABLE_EMPTY_WINDOW_OUTPUT_CONFIG):
+        result = _apply_post_sliding_window_process_function(
+            result,
+            window_descriptor,
+            agg_descriptors,
+            config.get(ENABLE_STATIC_WINDOW_OUTPUT_CONFIG),
+        )
+
```

Comment:
To improve code's readability, could you assert `ENABLE_STATIC_WINDOW_OUTPUT_CONFIG == True` here and explain why we don't need to apply the post process function?